weather.station.server/weather.station.server/Models/WeatherUpdate.cs

```diff
@@ -15,6 +15,6 @@ public class WeatherUpdate
         public DateTime TimeStamp { get; set; }
         public double TemperatureC { get; set; }
         public double Humidity { get; set; }
-	public double Windspeed { get; set; }
+	    public double Windspeed { get; set; }
```

Comment:
Nog even de tab hier weghalen en volledig door spaties vervangen

Reply:
Fixed in 
https://github.com/hro-sma/weather-station-server/pull/26/commits/b00a16cd4e3be2c4f835db7a5506970cee43c981